part-spec/component.json

```diff
@@ -28,7 +28,7 @@
 		},
 		"thermal": {
 			"description": "component temperature and thermal resistance information",
-			"$ref": "./common/thermals.json#/thermalProperties"
+			"$ref": "./common/thermal.json#/thermalProperties"
```

Comment:
I think this needs to be "thermal" instead of "thermalProperties" to match the change you made in the other file.

Otherwise, this looks good.

Reply:
You are totally right. Great catch, thanks!